.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: false
+
+language: php
+
+php:
+    - 7.2
+    - nightly
+
+before_install:
+    - phpenv config-rm xdebug.ini || true
+    - phpenv config-add .travis/php.ini
+
+install:
+    - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
+
+script:
+    - vendor/bin/phpunit
+
+jobs:
+    include:
+
+        - stage: Test
+          env: DEPENDENCIES=low
+          php: 7.2
+          install:
+              - travis_retry composer update --classmap-authoritative --no-suggest  --prefer-dist --prefer-lowest --prefer-stable
+
+        - stage: Code Quality
+          env: CODING_STANDARDS
+          script:
+              - vendor/bin/phpcs
```

Comment:
Like the multi-stage builds 👍 Does it automatically build a matrix of no-environment variables and environment variables?

Reply:
The normal matrix builds get include in the 'Test' stage.

Reply:
Not sure how to fit it in, but this stage could actually run in parallel with the rest of `Test`?

Reply:
Output of the command seems empty, can it be more verbose when run in CI?